scripts/ide-run-scss-lint-from-docker.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# This script will run scss-lint
+
+PROJECT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd)
+
+docker run --rm \
+  -v $PROJECT_DIR:/style-guide \
+  brainly/style-guide bash -c "scss-lint /style-guide/$1 -f JSON | sed 's/\/style-guide\///g'"
```

Comment:
I think no - as other than that we would have to mount all our folder and files to main directory of docker. But maybe @marcinb knows something more?